app/assets/javascripts/backbone/views/timeline_table_view.coffee

```diff
@@ -20,9 +22,18 @@ class MS.Views.TimelineTableView extends Backbone.View
 
   newGroup: (e) ->
     e.preventDefault()
-    @modal.show()
+    @modal.new()
+
+  newChildGroup: (e) ->
+    e.preventDefault()
```

Comment:
We see `e.preventDefault()` in each of these methods, but I don't know of a way to clean that up without making some questionable choices / unexpected behaviour.